src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -464,6 +464,7 @@ public class P2 { }");
             {
                 "using System.IO;",
                 "Rename 'P2' to 'Stream'",
+                "Generate type 'Foober'",
```

Comment:
test now validates that there should be the top level item, not just the list of child items.